backend/src/api/account/account.service.js

```diff
@@ -14,30 +14,43 @@ import { randomUUID } from 'crypto';
 import { CacheService } from '../../services/index.js';
 import { ACCOUNT_STATE } from '../../shared/enum/index.js';
 
-const { User, Account, Role } = sequelize;
+const { User, Account, Role, StudentClass } = sequelize;
 
 class AccountService {
     async getUserOfAccount(accountId) {
-        const user = await User.findAll({
-            where: {
-                userId: accountId
-            }
-        });
-        return user.map((x) => AccountDto.toDto(x));
+        try {
+          const user = await User.findOne({
+            where: { userId: accountId }
+          });
+          if (!user) {
+            throw new Error("User not found");
+          }
+          return AccountDto.toDto(user);
+        } catch (error) {
```

Comment:
remove try-catch in service, global exception will take care of this 👍, try-catch in controller instead